spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe User do
+  subject = User.new(name: 'franck', photo: 'https', bio: 'bio', posts_counter: 10)
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter must be greater than or equal to zero' do
+    subject.posts_counter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add tests for the `recent_post` methode in the user model.